test/e2e/network_test.go

```diff
@@ -452,7 +452,7 @@ func (tc *testContext) getWindowsServerContainerImage() string {
 		windowsServerImage = "mcr.microsoft.com/powershell:lts-nanoserver-2004"
 	} else if tc.CloudProvider.GetType() == config.AzurePlatformType {
 		// On Azure we are testing 20H2
-		windowsServerImage = "mcr.microsoft.com/powershell:lts-nanoserver-20h2"
+		windowsServerImage = "mcr.microsoft.com/powershell:lts-nanoserver-ltsc2022"
 	} else {
 		// For other providers we use 1809
 		windowsServerImage = "mcr.microsoft.com/powershell:lts-nanoserver-1809"
```

Comment:
I don't feel we will be using WS2019 as default any longer. PTAL at this [commit](https://github.com/openshift/windows-machine-config-operator/pull/1091/commits/49bab21938703c8afbb60f65674b239686f655f3#diff-84675b994e150b2fe4128b3f0a171854d1060ba909b9ff0aeb8833c8d9f4dae9R455), where I'm proposing WS2022 as the new normal.

Reply:
i dont see anything wrong with this change
rebase conflicts can be handled when they arise.